app/controllers/appeal-application/confirmation-page.ts

```diff
@@ -1,13 +1,14 @@
 import config from 'config';
 import { NextFunction, Request, Response, Router } from 'express';
 import i18n from '../../../locale/en.json';
-
-const daysToWaitAfterSubmission: number = config.get('daysToWait.afterSubmission');
-
 import { paths } from '../../paths';
 import { addDaysToDate } from '../../utils/date-utils';
 
+const daysToWaitAfterSubmission: number = config.get('daysToWait.afterSubmission');
+
 function getConfirmationPage(req: Request, res: Response, next: NextFunction) {
+  req.app.locals.logger.trace(`Successful appeal submission ${JSON.stringify(req.session.appeal.ccdCaseId)}`, 'Confirmation appeal submission');
```

Comment:
@gavinslilley can the text be updated to

`Successful AIP appeal submission for ccd id`